src/components/input/input.scss

```diff
@@ -52,11 +52,11 @@
   }
 
   /**
-   * Multi-line types should be taller
+   * Multi-line types get their height from the `rows` attribute
    */
 
   @at-root textarea#{&} {
-    height: sizes.$control-height-multiline;
+    height: auto;
```

Comment:
I wonder if it still makes sense to have a `min-height`? Otherwise a 1 row `textarea` may just look like an `input`? Though maybe that's okay?

Reply:
Does this have the correct height if `rows` is omitted or set to `1`?

Reply:
Oh that's a good question, actually. I was only thinking about the elastic textareas which will automatically get a min-height from `rows="2"`.

I'll restore the existing style here and just reset `height` for `.js-elastic-textarea`.

Reply:
I actually moved the style to a `c-input--elastic` modifier to better follow our BEM rules. JS-specific classnames don't belong in CSS files, by definition.